aim/agent/aid/universes/aci/converter.py

```diff
@@ -670,6 +743,20 @@ def infraRsVlan_vmm_id_converter(object_dict, otype, helper, to_aim=True):
         'skip': ['primary_addr_b', 'secondary_addr_a_list',
                  'secondary_addr_b_list'],
     }],
+    'bgpPeerP': [{
+        'resource': resource.L3OutInterfaceBgpPeerP,
+        'skip': ['asn', 'local_asn']
+    }],
+    'bgpAsP': [{
+        'resource': resource.L3OutInterfaceBgpPeerP,
+        'converter': bgp_as_converter,
```

Comment:
doesn't the default converter work here?

Reply:
We have to dealias here as I explained. I will remove the LocalAsnP conversion, but this one I think is necessary